src/containers/ProductListing.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch, useSelector } from 'react-redux';
+import { setProducts } from '../redux/actions/productsActions';
+import ProductComponent from './ProductComponent';
+
+const URL = 'https://fakestoreapi.com/products';
+
+const ProductPage = () => {
+  const products = useSelector((state) => state.allProducts.products);
+  const dispatch = useDispatch();
+  const getProducts = async () => {
+    const response = await axios
+      .get(URL)
+      .catch((err) => {
+        console.log('Err: ', err);
+      });
+    dispatch(setProducts(response.data));
+  };
+
+  useEffect(() => {
+    getProducts();
+  }, []);
+
+  console.log('Products :', products);
+  return (
+    <div className="ui grid container">
+      <ProductComponent />
+    </div>
```

Comment:
- [ ] I think you didn't link your details page with your list page so I can't  see any arrow could take me to the details page or when I press on thr card itself I can't see it could take me to the details page so kindly implement this functionality 🔥🚀💯

Reply:
- [ ] This issue is well resolved :heavy_check_mark: